pipelines/controller/src/test/resources/application-test.properties

```diff
@@ -19,4 +19,6 @@ fhirdata.incrementalSchedule=0 0 * * * *
 fhirdata.resourceList=Patient,Encounter,Observation
 fhirdata.dwhRootPrefix=config/controller_DWH
 fhirdata.purgeSchedule=0 30 * * * *
-fhirdata.numOfDwhSnapshotsToRetain=3
\ No newline at end of file
+fhirdata.numOfDwhSnapshotsToRetain=3
+fhirdata.fhirServerPassword=Admin123
+fhirdata.fhirServerUserName=Admin
```

Comment:
Note to myself: We should update the default/example configs like this [application.yaml](https://github.com/google/fhir-data-pipes/blob/master/docker/config/application.yaml).